pkg/finder/podresourcefinder.go

```diff
@@ -0,0 +1,147 @@
+package finder
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/davecgh/go-spew/spew"
+
+	"k8s.io/api/core/v1"
+
+	podresources "k8s.io/kubernetes/pkg/kubelet/apis/podresources"
+	podresourcesapi "k8s.io/kubernetes/pkg/kubelet/apis/podresources/v1alpha1"
+)
+
+const (
+	defaultPodResourcesTimeout = 10 * time.Second
+	defaultPodResourcesMaxSize = 1024 * 1024 * 16 // 16 Mb
+)
+
+type PodResourceFinder struct {
+	args              Args
+	podResourceClient podresourcesapi.PodResourcesListerClient
+	NodeResourceData  *NodeResources
```

Comment:
why public? (capital letter identifier)

Reply:
updating it to private, I am not sure why I made it public in the first place :)